kafka/proxy.go

```diff
@@ -60,6 +56,14 @@ func NewProxy(c *ProxyConfig) (proxy.Proxy, error) {
 		_ = server.Server.Flags().Set("log-level", "debug")
 	}
 
+	if c.TLS != nil && c.TLS.Enable {
```

Comment:
This was previously configured through the `c ExtraConfig` but now they have been moved into the config directly as it is important to be explicit.